src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java

```diff
@@ -152,13 +158,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
             player.getInventory().clear();
             player.setOp(false);
             player.setGameMode(GameMode.SURVIVAL);
-            plugin.pl.getPlayer(player).getFreezeData().setFrozen(true);
+            plugin.pl.getPlayer(player).setEditBlocked(true);
```

Comment:
Okay, then can I suggest raising a bug because this will be blocked by the fact you've changed this.